core/src/main/resources/hudson/lifecycle/WindowsInstallerLink/_restart.jelly

```diff
@@ -21,21 +21,44 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
-
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <st:statusCode value="503" /><!-- SERVICE NOT AVAILABLE -->
-  <l:layout permission="${app.ADMINISTER}">
```

Comment:
Checking permissions is not necessary any more?

Reply:
It's a restart page containing no information :) From my PoV it was a mistake to request ADMIN here.

Reply:
@daniel-beck are there any use-cases we might be missing?

Reply:
This view is intended to only be accessible to admins via https://github.com/jenkinsci/jenkins/blob/fdfcbc3c7f55746878000f2e4b5599abada7ea73/core/src/main/java/hudson/lifecycle/WindowsInstallerLink.java#L177-L187 so it seems like just the default to apply here.

OTOH, if there's nothing protection-worthy going on anyway?

Could it confuse people into thinking Jenkins is down by linking to this URL or something along those lines?

Reply:
As the page automatically redirects to the rootURL, it could be difficult to "stuck" people there.